WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -315,4 +322,33 @@ public void onClick(DialogInterface dialog, int which) {
             return builder.create();
         }
     }
+
+    /**
+     * dividers for sites
+     */
+    public static class SitePickerItemDecoration extends RecyclerView.ItemDecoration {
+        private Drawable mDivider;
```

Comment:
A minor point, but you can make this `private final` rather than just `private`.